proto/debugpb.proto

```diff
@@ -99,3 +103,22 @@ message ScanMvccResponse {
     bytes key = 1;
     kvrpcpb.MvccInfo info = 2;
 }
+
+message Failure {
+    enum Type {
+        INVALID = 0;
+        INJECT = 1;
+        RECOVER = 2;
+    }
+
+    string name = 1;
+    Type type = 2;
+    string actions = 3;
+}
+
+message FailPointRequest {
+    repeated Failure failures = 1;
```

Comment:
Should it be one request for one failure?

Let's consider failures `[A, B, C]`, we may set A and B successfully but C failed, so what should we do? Revert A and B then return error to client? But it is possible that A or B is already be triggered.

Reply:
Just one should be OK.